website/addons/evernote/serializer.py

```diff
@@ -0,0 +1,72 @@
+from website.addons.base.serializer import OAuthAddonSerializer
+from website.util import api_url_for, web_url_for
+
+class EvernoteSerializer(OAuthAddonSerializer):
```

Comment:
While not *technically* a "storage addon," this should probably be inheriting from `StorageAddonSerializer`, as `serialize_settings` is doing the same thing in both. You'll need to implement a `credentials_are_valid(self, user_settings, client)` method that checks to see if API calls will still work, and a `serialized_folder(self, node_settings)` that returns a JSON-formatted representation of the selected "NoteBook", but it should end up being simpler than what you're currently doing.